layouts/solo/main-ajax.html

```diff
@@ -40,16 +40,15 @@
   __TVPage__.config = __TVPage__.config || [];
   __TVPage__.config.push(
     {
+      settings: {{ $.Scratch.Get "settings" }},
       loginid: {{ .Site.Params.loginid }},
       id: "tvpembed-{{ .File.BaseFileName }}",
-      settings: {{ $.Scratch.Get "settings" }},
-      attributes: {
-        referenceIds: ["02","01"],
-        category: [{
-          value: "coffee",
-          label: "main_category"
-        }]
-      }
+      channelId : "84956291",
+      attributes: {             // All the attributes will go here. Put referenceId as comma seperated for multiple
+        referenceId: "02,03",
+        category_id_1: "123",
+        category_id_2: "123"
+      }    
```

Comment:
This shall be like this:

` attributes: {
          search: "### REPLACE WITH SEARCH QUERY",
          
          brand: ["### REPLACE WITH BRAND(S) ###"],
          
          category: [{
            value: "### REPLACE WITH CATEGORY ID ###",
            label: "### REPLACE WITH CATEGORY NAME ###"
          }],
       }`

This will be defined soon here https://tvpage.atlassian.net/wiki/display/DEV/Widget+Embed+Standards